projects/demo/src/pages/pages.ts

```diff
@@ -105,6 +105,12 @@ export const DEMO_PAGES: TuiDocPages = [
         route: DemoPath.DateTime,
         keywords: `date, day, month, year, mask, time, date-time, hour, minute, second, kit, generator`,
     },
+    {
+        section: 'Phone',
```

Comment:
What do you think about this idea ?

![Screenshot 2023-08-11 at 17 28 58](https://github.com/taiga-family/maskito/assets/35179038/b894b050-5ccb-44df-84b4-b3ebf66815be)

Reply:
https://maskito.dev/addons/phone